recipes/maxentscan/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: maxentscan
+  version: "1.0"
+source:
+  fn: fordownload.tar.gz
```

Comment:
Actually conda renames it for you if you specify another filename in fn. That is the purpose of fn.